CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## BETA
```

Comment:
I don't think is a BETA any more

Reply:
@donavanbecker please comment on the specifics, but i think this version in the changelog is updated _just_before merging into `latest` branch?

Reply:
Technically this is beta because we haven't release 4.50.7 and that is the next version.

Reply:
Maybe add the Verison number then beta after or test?

`4.50.-test.*`?

Or

`Beta 4.50.7`?

Reply:
@donavanbecker but we wouldn’t normally do this in a changelog file right?

Reply:
Normally do what? Create the change log? I would say yes we would create it while we push out betas to test.